packages/sysvars/src/recent-blockhashes.ts

```diff
@@ -0,0 +1,51 @@
+import { assertAccountExists, decodeAccount, type FetchAccountConfig } from '@solana/accounts';
+import { type Codec, getArrayCodec, getStructCodec, getU64Codec, mapCodec } from '@solana/codecs';
+import type { GetAccountInfoApi } from '@solana/rpc-api';
+import type { Rpc } from '@solana/rpc-spec';
+import { type Blockhash, getBlockhashCodec, lamports, type LamportsUnsafeBeyond2Pow53Minus1 } from '@solana/rpc-types';
+
+import { fetchEncodedSysvarAccount, SYSVAR_RECENT_BLOCKHASHES_ADDRESS } from './sysvar';
+
+type FeeCalculator = Readonly<{
+    lamportsPerSignature: LamportsUnsafeBeyond2Pow53Minus1;
+}>;
+type Entry = Readonly<{
+    blockhash: Blockhash;
+    feeCalculator: FeeCalculator;
+}>;
+
+/**
+ * The `RecentBlockhashes` sysvar.
+ *
+ * Information about recent blocks and their fee calculators.
+ */
+export type SysvarRecentBlockhashes = Entry[];
+
+export function getSysvarRecentBlockhashesCodec(): Codec<SysvarRecentBlockhashes> {
+    const lamportsCodec: Codec<LamportsUnsafeBeyond2Pow53Minus1> = mapCodec(
+        getU64Codec(),
+        val => val,
+        val => lamports(val),
+    );
```

Comment:
Since this is reused a few times, it might be good to extract it under the `@solana/rpc-types` package where the lamport type lives (with separate `Encoders` and `Decoders` ideally).

Reply:
Yeah I was thinking the same thing, but this opens a box to adding a codec for every type in that library that uses `unique symbol`. If you think it should be in there I'll add it.